ze/sampling_daemon.c

```diff
@@ -0,0 +1,803 @@
+#include "sampling_daemon.h"
+#include "../sampling/thapi_sampling.h"
```

Comment:
I think it should just be `thapi_sampling.h` and the autotool should just add `sampling`  to `-I` IMO.

What do you think @Kerilk ?